README.md

```diff
@@ -20,7 +20,7 @@ following example:
 
 ```hcl
 module "address-fe" {
-  source  = "terraform-google-modules/address/google"
```

Comment:
Hmm - Source should use the registry, so I think these need to match `terraform-google-modules/address/google`?

Reply:
Yup, here is an error from the linter:

```
terraform_validate ./examples/internal_with_dynamic_ip
╷
│ Error: Invalid registry module source address
│ 
│ Module "address" (declared at main.tf line 18) has invalid source address
│ "terraform-google-modules/terraform-google-address": a module registry
│ source address must have either three or four slash-separated components.
│ 
│ Terraform assumed that you intended a module registry source address
│ because you also set the argument "version", which applies only to registry
│ modules.
```